src/Altinn.App.Core/Features/Maskinporten/Models/TokenWrapper.cs

```diff
@@ -0,0 +1,49 @@
+using System.ComponentModel;
+using Altinn.App.Core.Models;
+
+namespace Altinn.App.Core.Features.Maskinporten.Models;
+
+/// <summary>
+/// JWT wrapper which includes the <see cref="AccessToken"/> and relevant metadata
+/// </summary>
+[ImmutableObject(true)] // `ImmutableObject` prevents serialization with HybridCache
+public sealed record TokenWrapper
```

Comment:
Hmm I'm a bit unsure still about this structure. Returning a wrapper from a public interface. What if `AccessToken` itself had these properties? They are all encoded inside the JWT anyway right?

Reply:
Feel like having a crack at something else here? I'm open to suggestions, but seeing as this was my second attempt, perhaps its time to hand over the reins 😄

Reply:
heres a proposal: https://github.com/Altinn/app-lib-dotnet/pull/915

Reply:
Makes sense, I like it! The name is awkward, because the "t" in "jwt" stands for "token, but I don't have any other good suggestions.

And people walk around saying stuff like "pin number" and "atm machine" all the time, so I doubt anyone besides me will care 😅

Reply:
Merged